cinema/serializers.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True, max_length=100)
+    duration = serializers.IntegerField(required=True)
```

Comment:
The `description` field from the `Movie` model is not included in the serializer. If you intend to handle the `description` field in API requests, consider adding it to the serializer.